testing/test_assertion.py

```diff
@@ -752,6 +752,51 @@ def test_dataclasses(self, testdir):
                 "*Omitting 1 identical items, use -vv to show*",
                 "*Differing attributes:*",
                 "*field_b: 'b' != 'c'*",
+                "*- c*",
```

Comment:
Perhaps we should also have a test for `attrs` to prevent any regressions there?